### PR TITLE
Feature/ch17837/asset magic urls

### PR DESCRIPTION
**Pull request by vuori777**
Adds support for `/asset` URLs and generating them.

Also linting.

### CODE REVIEW

File: skipscale/config.py [hunk 1]
```diff
@@ -100,10 +100,10 @@ def sentry_dsn(self) -> Optional[str]:
             return self.validated_config["sentry_dsn"]
         return None
 
-    def sentry_traces_sample_rate(self) -> Optional[float]:
+    def sentry_traces_sample_rate(self) -> float:
         if "sentry_traces_sample_rate" in self.validated_config:
             return self.validated_config["sentry_traces_sample_rate"]
-        return None
+        return 1.0
```
Comment (markonen) [inline skipscale/config.py:107]: The intent here was for tracing to be disabled unless `sentry_traces_sample_rate` was specified in the config.

File: skipscale/main.py [hunk 2]
```diff
@@ -16,7 +16,11 @@ async def healthcheck(request):
     return Response(status_code=200)
 
 routes = [
+    # Used for original images
     Route('/original/{tenant}/{image_uri:path}', original),
+    # Used for News assets
```
Comment (markonen) [inline skipscale/main.py:21]: Would not refer to News here, instead explain that it's for non-image assets that traverse skipscale just for the crypto infrastructure

**Review by markonen — CHANGES_REQUESTED** [review]
Just a couple of non-critical comments, LGTM otherwise!

File: skipscale/config.py (hunk 1 above)
Comment (vuori777) [reply skipscale/config.py:107]: typing sez that the sentry-sdk parameter is not Optional, changed to 0.0.

**Pull request merged**